File: Q7.java
import java.text.DecimalFormat;
import java.util.Scanner;

public class Q7 
{
    public static Scanner leia = new Scanner(System.in);

    public static void main(String args[]) 
    {
        float eleitores, brancos, nulos, validos, brancos_resultado, nulos_resultado, validos_resultado;
        String percentual_brancos, percentual_nulos, percentual_validos;
        System.out.print("Digite o total de eleitores no município: ");
        eleitores = leia.nextInt();
        System.out.print("Digite o total de votos brancos: ");
        brancos = leia.nextInt();
        System.out.print("Digite o total de votos nulos: ");
        nulos = leia.nextInt();
        System.out.print("Digite o total de votos válidos: ");
        validos = leia.nextInt();
        brancos_resultado = brancos * 100 / eleitores;
        nulos_resultado = nulos * 100 / eleitores;
        validos_resultado = validos * 100 / eleitores;
        percentual_brancos = new DecimalFormat("#.0").format(brancos_resultado);
        percentual_nulos = new DecimalFormat("#.0").format(nulos_resultado);
        percentual_validos = new DecimalFormat("#.0").format(validos_resultado);
        System.out.println("O percentual de votos, em relação ao total de eleitores no município, é:");
        System.out.print(percentual_brancos + "% votos brancos, ");
        System.out.print(percentual_nulos + "% votos nulos e ");
        System.out.println(percentual_validos + "% votos válidos.");
    }
}

File: Q7.cs
using System;

namespace Q7
{
    class Program
    {
        static void Main(string[] args)
        {
            int eleitores, brancos, nulos, válidos;
            float eleitoresf, brancosf, nulosf, válidosf, p_brancos, p_nulos, p_válidos;
            string pf_brancos, pf_nulos, pf_válidos;
            Console.Write("Digite o total de eleitores no município: ");
            eleitores = int.Parse(Console.ReadLine());
            eleitoresf = (float)eleitores;
            Console.Write("Digite o total de votos brancos: ");
            brancos = int.Parse(Console.ReadLine());
            brancosf = (float)brancos;
            Console.Write("Digite o total de votos nulos: ");
            nulos = int.Parse(Console.ReadLine());
            nulosf = (float)nulos;
            Console.Write("Digite o total de votos válidos: ");
            válidos = int.Parse(Console.ReadLine());
            válidosf = (float)válidos;
            p_brancos = brancosf * 100 / eleitoresf;
            p_nulos = nulosf * 100 / eleitoresf;
            p_válidos = válidosf * 100 / eleitoresf;
            pf_brancos = string.Format("{0:N1}", p_brancos);
            pf_nulos = string.Format("{0:N1}", p_nulos);
            pf_válidos = string.Format("{0:N1}", p_válidos);
            Console.WriteLine("O percentual de votos, em relação ao total de eleitores no município, é:");
            Console.WriteLine("{0}% votos brancos, {1}% votos nulos e {2}% votos válidos.", pf_brancos, pf_nulos, pf_válidos);
        }
    }
}